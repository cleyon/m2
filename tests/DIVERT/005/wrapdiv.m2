@define X 111
@newcmd leaving
We are leaving now, X=@X@
@endcmd
@newcmd bye
That's all folks!
@endcmd
@wrap @leaving
@wrap @bye
@divert 3
In stream 3, X is @X@
@divert
Start of file
@define X 222
In main, X=@X@
@divert 1
In stream 1, X is @X@
@divert
@define X 333
End of file
