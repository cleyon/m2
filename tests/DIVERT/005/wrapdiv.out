Start of file
In main, X=222
End of file
In stream 1, X is 222
In stream 3, X is 111
We are leaving now, X=333
That's all folks!
